index.js

```diff
@@ -12,16 +12,15 @@
  │   See the License for the specific language governing permissions and       │
  │   limitations under the License.                                            │
  \*───────────────────────────────────────────────────────────────────────────*/
-'use strict';
```

Comment:
I vaguely remember running into a const/let error in node v4 when removing a use strict, but I can't seem to reproduce it.  As long as things are still working in v4, this seems fine to me.